pages/linux/ip-route.md

```diff
@@ -0,0 +1,32 @@
+# ip route
+
+> IP Routing table management subcommand.
+> More information: <https://www.man7.org/linux/man-pages/man8/ip-route.8.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/ip-route>.
```
Maybe we should use the manned.org page here. (https://github.com/tldr-pages/tldr/issues/5143#issuecomment-812939570)